src/workflows-app/SubmissionDetails.test.js

```diff
@@ -553,4 +553,140 @@ describe('Submission Details page', () => {
 
     expect(screen.getByText('Some workflow statuses are not up to date. Refreshing the page may update more statuses.')).toBeInTheDocument();
   });
+
+  it('should display inputs on the Inputs tab', async () => {
+    const user = userEvent.setup();
+    const getRuns = jest.fn(() => Promise.resolve(runsData));
+    const getRunsSets = jest.fn(() => Promise.resolve(runSetData));
+    const getMethods = jest.fn(() => Promise.resolve(methodData));
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+    Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runs: {
+            get: getRuns,
+          },
+          runSets: {
+            get: getRunsSets,
+          },
+          methods: {
+            getById: getMethods,
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      render(
+        h(BaseSubmissionDetails, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+          submissionId,
+        })
+      );
+    });
+
+    const workflowsTabButton = screen.getByRole('button', { name: 'Workflows' });
+    expect(workflowsTabButton !== undefined);
+
+    const inputsTabButton = screen.getByRole('button', { name: 'Inputs' });
+    expect(inputsTabButton !== undefined);
+
+    // ** ACT **
+    // user clicks on inputs tab button
+    await user.click(inputsTabButton);
+
+    expect(screen.getByRole('table')).toBeInTheDocument();
+
+    const inputTable = screen.getByRole('table');
+    const rows = within(inputTable).getAllByRole('row');
+    expect(rows.length).toBe(JSON.parse(runSetData.run_sets[0].input_definition).length + 1); // one row for each input definition variable, plus headers
+
+    const headers = within(rows[0]).getAllByRole('columnheader');
+    expect(headers.length).toBe(5);
+
+    const row1cells = within(rows[1]).getAllByRole('cell');
+    expect(row1cells.length).toBe(5);
+    expect(row1cells[0].textContent).toBe('hello');
```

Comment:
FYI: `@testing-library/jest-dom` includes several DOM related matchers which can make tests clearer to read.

```suggestion
    expect(row1cells[0]).toHaveTextContent('hello');
```

https://github.com/testing-library/jest-dom?tab=readme-ov-file#table-of-contents